code/modules/research/designs/medical_designs.dm

```diff
@@ -62,6 +62,16 @@
 	build_path = /obj/item/handheld_defibrillator
 	category = list("Medical")
 
+/datum/design/holostretcher
+	name = "Holo Strecher"
+	desc = "A hardlight projector for transporting patients."
+	id = "holo_stretcher"
+	req_tech = list("biotech" = 5, "magnets" = 4, "powerstorage" = 4)
```

Comment:
biotech feels weird for this, it involves zero biology? maybe replace it by upping the magnets to 5? 6?